### PR TITLE
Examples: Clean up.

### PR DESCRIPTION
**Pull request by Mugen87**
Related issue: -

**Description**

Some linter fixes.

### CODE REVIEW

**Comment by mrdoob** [comment]
Sorry, merging #21282 created conflicts here.

**Comment by Mugen87** [comment]
Np. Done!

**Comment by mrdoob** [comment]
Thanks!

File: utils/modularize.js [hunk 1]
```diff
@@ -193,7 +193,6 @@ var files = [
 	{ path: 'shaders/LuminosityShader.js', dependencies: [], ignoreList: [] },
 	{ path: 'shaders/MirrorShader.js', dependencies: [], ignoreList: [] },
 	{ path: 'shaders/NormalMapShader.js', dependencies: [], ignoreList: [] },
-	{ path: 'shaders/OceanShaders.js', dependencies: [], ignoreList: [] },
```
Comment (mrdoob) [inline utils/modularize.js:196]: Ops!

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**